### DISCUSSION TITLE
budibase-comment-box how to get the rowid?

### DISCUSSION THREAD

**Discussion by Sagestum** [Budibase/budibase#11057] [Help]
Hi there, 

I want to use the budibase-comment-box plugin and I need to give it a rowid. It sounds very trivial, but I can't get it to work. Where can I get the rowid?

https://github.com/aptkingston/budibase-comment-box
![image](https://github.com/Budibase/budibase/assets/11883823/36ca4c55-afb2-41fd-8556-4b318f6cefb7)
thanks

**Comment by andz-bb** [comment]
Hey @Sagestum 

You can find the Row ID for a given row by going to the relevant table in the data section. Right click on the row you want to use and you'll see an option to Copy the Row _id as shown below. You can then use this as the ID for your comment-box component. If you run into any more problems, let me know and I'll be happy to help!

![row _id](https://github.com/Budibase/budibase/assets/110921612/17c244b1-d6c2-4bc7-95db-c1d6b0b04726)

**Comment by Sagestum** [comment]
Should this not be the ID of the new row in a binding? If I set it to an id from the existing lines, it ends in a failure. 
![image](https://github.com/Budibase/budibase/assets/11883823/2ec94567-3ec3-40c0-807e-914655bd316a)

**Reply by spitoutzi** [reply]
did you manage to solve the problem? It would be logical to assume that you need to specify the ID of the record for which the comment is being created. For example, the ID of an appointment with a doctor in another table. I was creating a separate table with comments and a text field. In the ID, I selected the ID through the repeater, but the same error. I tried to create a relationship between tables, specify the ID from the SQL table, but constantly or an error as in the comment above or nothing happens at all. I tried to create a internal table, no matter what you specify, there is always an error

![image](https://github.com/Budibase/budibase/assets/126158687/456eed2f-00ba-4a22-b75b-174c68eada5f)

**Comment by spitoutzi** [comment]
have the same problem, using Postgres SQL. What ID I have to use?

**Reply by spitoutzi** [reply]
It worked! I created an empty row in a separate table with comments, copied its "_ID" and inserted it into the "Row ID" field. After that, everything worked out! It is also possible to get "_ID" through a repeater, for example. There is no need to create a separate table, just add a text field to an existing table. Maybe it will be useful :)
